NSS-Day-Cohort-61/team-1-GiffyGram
# Cjh favorites

#### Changes Made
    If you log out, things function a little more as you might expect.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
